http-server-jetty/src/test/groovy/io/micronaut/servlet/jetty/JettyJsonStreamSpec.groovy

```diff
@@ -152,11 +150,11 @@ class JettyJsonStreamSpec extends Specification {
         // Funny request flow which required the server to relase the semaphore so we can keep sending stuff
         def stream = client.jsonStream(HttpRequest.POST(
                 '/jsonstream/books/count',
-                Flowable.fromCallable {
+                Mono.fromCallable {
                     JettyJsonStreamSpec.signal.acquire()
                     new Book(title: "Micronaut for dummies")
                 }
-                        .repeat(10)
+                        .repeat(9)
```

Comment:
Just curious about why changing it to 9 repetitions instead of 10

Reply:
RxJava2 `Flowable.repeat` (http://reactivex.io/RxJava/2.x/javadoc/io/reactivex/Flowable.html#repeat-long-) says `repeat` "returns a Flowable that repeats the sequence of items emitted by the source Publisher at most count times... a count of 0 will yield an empty sequence."

Reactor `Flux.repeat` (https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Flux.html#repeat-long-) says `repeat` will "Repeatedly subscribe to the source numRepeat times. This results in numRepeat + 1 total subscriptions to the original source. As a consequence, using 0 plays the original sequence once."

Similar API, but off by one in both docs and experienced behavior.

Reply:
I saw the same in core.